roles/deploy/tasks/main.yml

```diff
@@ -17,6 +17,11 @@
 - import_tasks: update.yml
 - import_tasks: prepare.yml
 - import_tasks: build.yml
+
+- name: "Stop here if this is a dry-run"
+  meta: end_play
```

Comment:
TIL! Had no idea about the `meta` module 🆒

Reply:
Can you let me know what the output of this looks like as well? I'm wondering if it's useful to have a separate more descriptive message or if this is good enough.

Reply:
GitHub's output ends after the previous task:
<img width="843" alt="image" src="https://user-images.githubusercontent.com/2457670/229949811-4ef43d93-3440-424a-9f6a-fe217f77a443.png">

But locally it shows the following:
```
TASK [deploy : include_tasks] **************************************************
included: /path/to/example.com/trellis/roles/deploy/hooks/build-after.yml for example.com => (item=/path/to/example.com/trellis/roles/deploy/hooks/build-after.yml)

TASK [deploy : Check for composer.json in project root or project_subtree_path] ***
ok: [example.com]

TASK [deploy : Fail if composer.json not found] ********************************
skipping: [example.com]

TASK [deploy : Setup composer authentications] *********************************
ok: [example.com] => (item=None)
ok: [example.com]

TASK [deploy : Run composer check] *********************************************
changed: [example.com]

TASK [deploy : Install Composer dependencies] **********************************
changed: [example.com]

TASK [deploy : Stop here if this is a dry-run] *********************************

PLAY RECAP *********************************************************************
example.com : ok=26   changed=12   unreachable=0    failed=0    skipped=26   rescued=0    ignored=0   
localhost                  : ok=0    changed=0    unreachable=0    failed=0    skipped=1    rescued=0    ignored=0   


```